### PR TITLE
Event Tests

### PR DESCRIPTION
**Pull request by AydanPirani**
Will contain comprehensive tests for the events service

### CODE REVIEW

File: src/services/event/event-router.test.ts [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+            description: "Example Location",
+            tags: ["SIEBEL0", "ECEB1"],
+            latitude: 40.1138,
+            longitude: -88.2249,
+        },
+    ],
+    sponsor: "Example sponsor",
+    eventType: "OTHER",
+    points: 0,
+};
```
Comment (Timothy-Gonzalez) [inline src/services/event/event-router.test.ts:32]: np: Use satisfies to verify correctness

File: src/services/event/event-router.test.ts [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+    });
+});
+
+describe("PUT /event/metadata/", () => {
+    it("cannot be accessed by attendee", async () => {
+        const response = await putAsAttendee(`/event/metadata/`).expect(StatusCode.ClientErrorForbidden);
+        expect(response).toHaveProperty("error");
+    });
+
+    it("cannot be accessed by staff", async () => {
```
Comment (Timothy-Gonzalez) [inline src/services/event/event-router.test.ts:209]: don't need to test for every level - just test the level below what is expect (staff for admin only, attendee for staff only, etc)

**Review by Timothy-Gonzalez — CHANGES_REQUESTED** [review]
Not giving a formal review yet because this is still in progress, but here's two small things for now. General heuristic for tests is look at coverage and see what isn't covered - and make sure you're covering those edge cases.
